src/frontend/src/components/ProjectSubmissions/SubmissionsTable.tsx

```diff
@@ -127,20 +120,13 @@ const SubmissionsTable = ({ toggleView }) => {
       SubmissionFormFieldsService(`${import.meta.env.VITE_API_URL}/submission/submission_form_fields/${projectId}`),
     );
     dispatch(SubmissionActions.SetSubmissionTableRefreshing(true));
-    if (!filter.task_id) {
+    {
```

Comment:
Remove this opening and its corresponding closing curly bracket

Reply:
oh okay forgot extra bracket.